rules/common/src/main/kotlin/com/twitter/compose/rules/ComposeNaming.kt

```diff
@@ -14,11 +15,18 @@ class ComposeNaming : ComposeKtVisitor {
     override fun visitComposable(function: KtFunction, autoCorrect: Boolean, emitter: Emitter) {
         // If it's a block we can't know if there is a return type or not from ktlint
         if (!function.hasBlockBody()) return
-        val firstLetter = function.name?.first() ?: return
+        val functionName = function.name?.takeUnless(String::isEmpty) ?: return
```

Comment:
can the function name ever be empty 🤔

Reply:
Tbh I'm not sure, I was thinking to just be extra careful as I have no idea what happens with local functions